src/agent/CommandStatus.ts

```diff
@@ -79,8 +79,6 @@ class CommandStatus extends CommandPolykey {
               clientPort: response.clientPort,
               agentHost: response.agentHost,
               agentPort: response.agentPort,
-              publicKeyJWK: response.publicKeyJwk,
-              certChainPEM: response.certChainPEM,
```

Comment:
Didn't you say you were working on something to expand the `agent status` command to add additional details like number of connections, number of nodes in the graph, number of gestalt vertices, and sigchain claims? Where is that PR?

Reply:
That is in a new branch called feature-cli-polish-2, since the expansion is still in discussion, I didn't want to muddy this PR.

I will be creating a PR for that soon.